talent/onboarding/onboarding.md

```diff
@@ -0,0 +1,68 @@
+
+## Onboarding Process
+
+- This document describes the onboarding process for a new source{d} member.
+
+### Guide 
+
+- Read the source{d} [guide](https://github.com/src-d/guide) which contains all the information necessary from the moment you accept an offer. 
+
+### Hardware
+
+- A few days before you start working with us, you choose the hardware you would like to use from on the [list](https://github.com/src-d/guide/blob/master/general/available_hardware.md) with the hardware we provide. 
+- The first day we present it to you, so you are able to create your working environment as soon as possible.
+
+### Accounts 
+
+- From the first day you have access to source{d} Google Account, while receiving invitations to join the applications we are using:
+* Slack: the main communication channel we are using. Click [here](https://github.com/src-d/guide/blob/master/communication/slack_channels.md) for more information. 
+* GitHub source{d}: you are added to our GitHub organization. The lead of your team is responsible to explain the repositories you need to be notified of and how to treat them.
+* Lever: it's a platform to support our hiring process. Everyone has access to it in order to be able to review and give feedback on candidates. 
+* Kin HR: if you want to ask days off that's the right place to do it. Also, every month you will receive the invoice of your payment through the platform. 
+* Google Calendar: here are hosted all the events. Give to our Office Manager access to your calendar in order to add the events to it. Make sure you check your calendar frequently. 
+
+- The above are the main tools we are using, but we do have a full [list](https://github.com/src-d/guide/blob/master/general/tools.md).
+
+### Welcome Talks 
+
+- During your first week you are going to have Welcome Talks in order to get to know source{d} and every team better:
+* Welcome talks with the Founders
+* Welcome talk with the CTO
+* Welcome talk with the Product Manager
+
+### Office
+
+- At the office, we provide:
+
+* Kitchen with refrigerator and microwaves. Coffee, milk, fruits, drinks, ice creams etc.
+* HTC Vive VR Headset, Xbox One and Nintendo 64 with a nice catalog of games. 
+* Boxes to store your personal items.
+
+- In case you don't know how to use any of the electronics feel free to ask the person who might there with you. 
+- For any issues you may notice in terms of condition or lacking supplies, please contact our Office Manager.
```

Comment:
is opening [issue in office repo](https://github.com/src-d/office/issues) the recommended way @estherrgarcia ?